Makefile

```diff
@@ -55,9 +77,23 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: header.html $(DOC_FILES) $(FIGURE_FILES)
 endif
 
 header.html: .tool/genheader.go specs-go/version.go
+	# create a symlink to opencontainers if we are running in a fork
```

Comment:
This isn't necessary to include in the Makefile, this is not something that is generally supported or encouraged by packages

Reply:
Taken care of